project/Dependencies.scala

```diff
@@ -70,7 +70,8 @@ object Dependencies {
   val kafka_test = kafka % "test" classifier "test"
   val mockito_test = "org.mockito" % "mockito-core" % versions.mockito % "test"
   // Functionally Nice REST Client Currently newman only supports scala version 2.9.2 and 2.10
-  val newman = "com.stackmob" %% "newman" % versions.newman
+  // Because of this, its been replaced with megam newman, which supports scala 2.11
```

Comment:
I wrote an improved comment here. It's gone now. If you are going to force push a change, you need to make sure you are up to date first.